packages/x-data-grid/src/components/GridPagination.tsx

```diff
@@ -48,7 +48,8 @@ const defaultLabelDisplayedRows: WrappedLabelDisplayedRows = ({ from, to, count,
   if (!estimated) {
     return `${from}–${to} of ${count !== -1 ? count : `more than ${to}`}`;
   }
-  return `${from}–${to} of ${count !== -1 ? count : `more than ${estimated > to ? estimated : to}`}`;
+  const estimateLabel = estimated && estimated > to ? `around ${estimated}` : `more than ${to}`;
```

Comment:
Would this change be breaking for some users?
I imagine people writing test cases for their use cases based on a textual value.
How about extracting this change to be on the safe side?

Reply:
Extracted here
https://github.com/mui/mui-x/pull/15632
I will revert changes in this PR